### ISSUE TITLE
Computing accuracy for the mixup

### ISSUE THREAD

**Issue by aluex** [tugstugi/pytorch-speech-commands#5]
Currently in the code _train_cifar10.py_

```
if args.mixup:
            _, targets = batch
            targets = Variable(targets, requires_grad=False).cuda(non_blocking=True)
correct += pred.eq(targets.data.view_as(pred)).sum()
```

I don't see why the accuracy is comparing the outputs of **mixed samples** with the **original** y labels. They simply do not match.

**Comment by tugstugi** [comment]
Yes, this is wrong. But what you are normally interested in is the validation/test accuracy which should be correct because no mixup applied there.
